semana3/projeto-blackjack/desafio.js

```diff
@@ -0,0 +1,48 @@
+console.log ("Bem vindo ao jogo de BlackJack!");
+if(confirm("Deseja iniciar uma nova rodada?")){
+   let pontosUsuario=0, pontosPc=0, cartasUsuario=[], cartasPc=[], comparador=0;
+   while(comparador===0){
+      for(let i=0; i<2; i++){
+            let carta = comprarCarta(); 
+            cartasUsuario.push(carta.texto)
+            pontosUsuario += carta.valor;
+            carta = comprarCarta(); 
+            cartasPc.push(carta.texto)
+            pontosPc += carta.valor;
+      }
+      if(pontosPc <= 21 || pontosPc <= 21){
+         comparador=1;
+      }
```

Comment:
Boa validação, mas ela também impede de começar com uma mão perfeita (com a soma indo até 21), e checa duas vezes a pontuação do computador, mas nenhuma a do usuário.

Reply:
Falta de atenção, era pra uma das verificações ser pontosUsuario, mas fora isso, essa variável vai fazer sair do while se a pontuação for igual a 21